### PR TITLE
various small fixes

### PR DESCRIPTION
**Pull request by MaximilianKrambach**
- motion detail comments not always visible
- fix tag/recommendation multiselect
- fix display of prefixed categories in filter/dialogs
- confirm dialog on email invitation
- attempt at fixing history filter

### CODE REVIEW

**Comment by MaximilianKrambach** [comment]
@emanuelschuetze for your last comment, the rest is review @tsiegleauq

File: client/src/app/site/users/components/user-list/user-list.component.ts [hunk 1]
```diff
@@ -258,8 +258,13 @@ export class UserListComponent extends ListViewBaseComponent<ViewUser> implement
      * Handler for bulk sending e-mail invitations. Uses selectedRows defined via
      * multiSelect mode.
      */
-    public sendInvitationEmailSelected(): void {
-        this.repo.sendInvitationEmail(this.selectedRows).then(this.raiseError, this.raiseError);
+    public async sendInvitationEmailSelected(): Promise<void> {
+        const content =
```
Comment (tsiegleauq) [inline client/src/app/site/users/components/user-list/user-list.component.ts]:
```
const content = `${this.translate.instant('Send invitation e-Mails to the selected users?')} (${this.selectedRows.length} E-Mails)`;
```

**Comment by emanuelschuetze** [comment]
Still open:
* comment box is still visible if admin added and removed content (=empty string)

File: client/src/app/site/users/components/user-list/user-list.component.ts (hunk 1 above)
Comment (MaximilianKrambach) [reply client/src/app/site/users/components/user-list/user-list.component.ts]: this turns aesthetically unpleasing as soon as the 'prettifier' stuff runs over it.

**Comment by MaximilianKrambach** [comment]
comment box now checks for both (not existing at all and being empty)

**Pull request merged**